<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="4" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-12-22T15:06:53 EST" name="RestAPISuite" finished-at="2020-12-22T15:06:57 EST" duration-ms="4385">
    <groups>
    </groups>
    <test started-at="2020-12-22T15:06:53 EST" name="EmployeesRestAPITest" finished-at="2020-12-22T15:06:57 EST" duration-ms="4385">
      <class name="com.restapi.testCases.TC004_GET_PrintAllHeaders">
        <test-method signature="getSingleemployeesDetail()[pri:0, instance:com.restapi.testCases.TC004_GET_PrintAllHeaders@c03cf28]" started-at="2020-12-22T15:06:56 EST" name="getSingleemployeesDetail" finished-at="2020-12-22T15:06:57 EST" duration-ms="608" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getSingleemployeesDetail -->
      </class> <!-- com.restapi.testCases.TC004_GET_PrintAllHeaders -->
      <class name="com.restapi.testCases.TC002_POST_Request">
        <test-method signature="createNewRecord()[pri:0, instance:com.restapi.testCases.TC002_POST_Request@55f616cf]" started-at="2020-12-22T15:06:54 EST" name="createNewRecord" finished-at="2020-12-22T15:06:56 EST" duration-ms="1151" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createNewRecord -->
      </class> <!-- com.restapi.testCases.TC002_POST_Request -->
      <class name="com.restapi.testCases.TC003_GET_Request">
        <test-method signature="getSingleemployeesDetail()[pri:0, instance:com.restapi.testCases.TC003_GET_Request@1356d4d4]" started-at="2020-12-22T15:06:56 EST" name="getSingleemployeesDetail" finished-at="2020-12-22T15:06:56 EST" duration-ms="626" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [application/json] but found [text/html; charset=UTF-8]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [application/json] but found [text/html; charset=UTF-8]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.restapi.testCases.TC003_GET_Request.getSingleemployeesDetail(TC003_GET_Request.java:39)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getSingleemployeesDetail -->
      </class> <!-- com.restapi.testCases.TC003_GET_Request -->
      <class name="com.restapi.testCases.TC005_GET_ExtractValuesOfEachNodeInJSON">
        <test-method signature="getemployeesDetails()[pri:0, instance:com.restapi.testCases.TC005_GET_ExtractValuesOfEachNodeInJSON@1329eff]" started-at="2020-12-22T15:06:57 EST" name="getemployeesDetails" finished-at="2020-12-22T15:06:57 EST" duration-ms="248" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getemployeesDetails -->
      </class> <!-- com.restapi.testCases.TC005_GET_ExtractValuesOfEachNodeInJSON -->
      <class name="com.restapi.testCases.TC001_GET_Request">
        <test-method signature="getemployeesDetails()[pri:0, instance:com.restapi.testCases.TC001_GET_Request@609db43b]" started-at="2020-12-22T15:06:53 EST" name="getemployeesDetails" finished-at="2020-12-22T15:06:54 EST" duration-ms="1697" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getemployeesDetails -->
      </class> <!-- com.restapi.testCases.TC001_GET_Request -->
    </test> <!-- EmployeesRestAPITest -->
  </suite> <!-- RestAPISuite -->
</testng-results>
